experimental/generation/generator/templates/professional/professional.schema

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
```

Comment:
I don't think you need this.

Reply:
Still don't need this.

---
In reply to: [435642984](https://github.com/microsoft/BotBuilder-Samples/pull/2474#discussion_r435642984) [](ancestors = 435642984)